src/qibolab/platform.py

```diff
@@ -10,12 +10,14 @@
 import yaml
 from qibo.config import log, raise_error
 
+from qibolab import ExecutionParameters
```

Comment:
Here it should be
```suggestion
from qibolab.execution_parameters import ExecutionParameters
```
right?

Reply:
I wrote both by mistake actually.